readme.txt

```diff
@@ -0,0 +1,34 @@
+=== Polylang translated table example ===
+Contributors: Chouby, manooweb, marianne38, greglone, hugod
+Donate link: https://polylang.pro
+Requires at least: 5.8
+Tested up to: 6.2
+Requires PHP: 5.6
+Stable tag: 1.0
+License: GPLv3 or later
+License URI: https://www.gnu.org/licenses/gpl-3.0.html
+
+This is an example plugin meant for developers and demonstrating how to translate a custom DB table with [Polylang](https://github.com/polylang/polylang/) 3.4.
+
+== Description ==
```

Comment:
As the description is the copy of the readme.md, maybe you should link to this file to avoid to maintain 2 files.

Reply:
I'm not even sure that we need this file.

Reply:
Actually I just discovered that the "View plugin details" link is not displayed for non-repository plugins, making this file useless, so I'll remove it.